File: Modul6/PRAK603-2310817110008-Muhammad_Raihan.c
#include <stdio.h>

void main(){
    int n1,n2;
    scanf("%d %d", &n1, &n2);
    if(n1!=n2){
        printf("Jumlah tidak sama");
    }
    else{
        int baris1[n1];
        int baris2[n2];
        for(int i=0; i<n1; i++){
            scanf("%d", &baris1[i]);
        }
        for(int i=0; i<n2; i++){
            scanf("%d", &baris2[i]);
        }
        printf("\n");
        for(int i=0; i<n1; i++){
            printf("%d ", baris1[i]*baris2[i]);
        }
    }
}

File: Modul6/PRAK603-2310817110008-Muhammad_Raihan.py
n1,n2 = map(int, input().split())
if n1!=n2:
    print("Jumlah tidak sama")
else:
    baris1 = list(map(int, input().split()))
    baris2 = list(map(int, input().split()))
    print("")
    for i in range(0,n1,1):
        print(baris1[i]*baris2[i],end=' ')